src/main/resources/META-INF/definitions.cnd

```diff
@@ -8,6 +8,7 @@
  - sitemapIndexURL (string) = ''  indexed=no hidden autocreated
  - sitemapHostname (string) = ''  indexed=no hidden autocreated
  - sitemapCacheDuration (string) = ''  indexed=no hidden autocreated
+ - isJobTriggered (boolean)  indexed=no hidden
```

Comment:
Do we use this property somewhere?

Reply:
no, it was for testing purpose, thank you!